src/futil.js

```diff
@@ -0,0 +1,6 @@
+import _ from 'lodash/fp'
+import F from 'futil'
+
+export let flattenProp = _.curry((prop, target) =>
```

Comment:
just for future reference, there are test cases in here when we move to futil: https://runkit.com/daedalus28/flattenprop